### PR TITLE
Add duration of element.longPress for iOS

### PR DESCRIPTION
**Pull request by jhen0409**
Related to #410. I also added the test case `Long Press Me 1.5s`.

### CODE REVIEW

File: detox/src/ios/expect.js [hunk 1]
```diff
@@ -58,9 +58,13 @@ class TapAtPointAction extends Action {
 }
 
 class LongPressAction extends Action {
-  constructor() {
+  constructor(duration) {
     super();
-    this._call = invoke.callDirectly(GreyActions.actionForLongPress());
+    if (duration) {
```
Comment (rotemmiz) [inline detox/src/ios/expect.js]: let's add default value, drop the condition, and use only `GreyActions.actionForLongPressWithDuration`

File: detox/test/e2e/c-actions.js [hunk 2]
```diff
@@ -17,13 +17,18 @@ describe('Actions', () => {
     await expect(element(by.text('Long Press Working!!!'))).toBeVisible();
   });
 
+  it(':ios: should long press with duration on an element', async () => {
```
Comment (rotemmiz) [inline detox/test/e2e/c-actions.js]: I don't want to create more discrepancy between iOS and Android API, I'm willing to help you add Android support as well if you need help.

File: detox/test/e2e/c-actions.js [hunk 3]
```diff
@@ -17,13 +17,18 @@ describe('Actions', () => {
...
+  });
+
   it('should multi tap on an element', async () => {
     await element(by.id('UniqueId819')).multiTap(3);
     await expect(element(by.id('UniqueId819'))).toHaveText('Taps: 3');
   });
 
   it('should tap on an element at point', async () => {
-    await element(by.id('View7990')).tapAtPoint({x:180, y:140});
+    await element(by.id('View7990')).tapAtPoint({x:180, y:160});
```
Comment (rotemmiz) [inline detox/test/e2e/c-actions.js]: Why was that changed ?

Comment (jhen0409) [reply detox/test/e2e/c-actions.js]: I added [a button for new test case](https://github.com/wix/detox/pull/412/files#diff-bed70c67a493bc438deb1689cadd32ee), so the `UniqueId819` button position was changed.

Comment (rotemmiz) [reply detox/test/e2e/c-actions.js]: 👍

File: detox/test/e2e/c-actions.js (hunk 2 above)
Comment (jhen0409) [reply detox/test/e2e/c-actions.js]: Yeah I also really want this feature can be supported for Android, if you could help will be great. Thanks. :)

File: generation/earl-grey/index.js [hunk 4]
```diff
@@ -47,6 +47,7 @@ if (
 
 // Constants
 const SUPPORTED_TYPES = [
+  "CFTimeInterval",
```
Comment (DanielMSchmidt) [inline generation/earl-grey/index.js]: 🎉🎉🎉

**Comment by jozan** [comment]
How is this progressing? Can I help somehow? We pretty much need this feature for both platforms. :)

**Comment by jozan** [comment]
Alright, I started working on this feature base on this PR.

**Comment by rotemmiz** [comment]
@jozan, for Android?

**Comment by jozan** [comment]
For iOS, we need this feature eventually for Android as well but even
getting iOS to work would be very nice.

I tried to rebase to master but it did not work out of the box since lot
has changed past three months especially in generation dir. Also, after
rebase both longPress and longPress(duration) did not pass tests and failed
to error from this line:
https://github.com/wix/detox/blob/master/detox/ios/Detox/MethodInvocation.m#L225
where value i was 0.

I don’t have time to debug further today.

**Comment by rotemmiz** [comment]
Go away stale bot! Sorry for the huge delay. We have intention to merge this in.

**Comment by LeoNatan** [comment]
Any news on this PR? Let’s merge for iOS and then in the future add for Android?

**Comment by rotemmiz** [comment]
Yes, let's do it

**Comment by LeoNatan** [comment]
@jhen0409 Could you please merge master and see that tests are not broken?

**Comment by jhen0409** [comment]
@LeoNatan 👌

**Comment by LeoNatan** [comment]
Thanks!

**Comment by LeoNatan** [comment]
@yershalom Jenkins is flaky, please look why.

File: detox/src/ios/expect.js [hunk 5]
```diff
@@ -65,9 +65,9 @@ class TapAtPointAction extends Action {
 }
 
 class LongPressAction extends Action {
-  constructor() {
+  constructor(duration = 750) {
```
Comment (LeoNatan) [inline detox/src/ios/expect.js]: Why is the default 750? Is this what Earl Grey uses in the long press action?

Comment (jhen0409) [reply detox/src/ios/expect.js]: I forgot the reason, but it should be 1000 for now.

Comment (LeoNatan) [reply detox/src/ios/expect.js]: Let's instead use the default. If no param is provided, let's call the current API, but if there is provided, let's call the new API.

**Comment by jhen0409** [comment]
The some style changes caused by Prettier, I reverted it.

**Comment by LeoNatan** [comment]
Cool. Let’s wait for the CI and I’ll merge. Thank you for your contribution!
